index.js

```diff
@@ -430,6 +427,82 @@ instance.prototype.updateScenesAndSources = async function() {
 		}
 	}
 
+	let updateSceneSources = (source, scene) => {
```

Comment:
Wouldn't this function be easier to read of you'd store `self.sources[source.name]` in a `const`?

Reply:
I'm not quite sure how to do this. I tried declaring this within each block, but it seems to keep ending up in a TypeError with the const being undefined.

Reply:
I think the problem I'm running into if I try and do this is that sources are not always defined yet when running through this process, so some aren't assigned to self.sources until later in the process.